File: Exercícios/Exercicios/src/exercicios_java/Ex27.java
package exercicios_java;

import java.util.Scanner;

public class Ex27 {

	public static void main(String[] args) {
		//  Faça um algoritmo que leia uma variável e some 5 caso seja par ou some 8 
		// caso seja ímpar, imprimir o resultado desta operação.
		
		Scanner ler = new Scanner (System.in);
		int a, b, res;
		System.out.printf("Digite um valor: ");
		a = ler.nextInt();
		
		if (a % 2 == 0) {
			res = a + 5;
			System.out.printf("O valor digitado é par, somado com 5 dá %d", res);
		} else {
			res = a + 8;
			System.out.printf("O valor digitado é ímpar, somado com 8 dá %d", res);
		}
		
		
		
		
		
		
		
		
	}

}

File: Python/Exercicios/Ex27.py
# somar 5 se for par ou somar 8 se for impar

a = int(input("Digite um valor: "))

if (a % 2 == 0):
    res = a + 5
    print(f"O número digitado é par, somado com cindo dá : {res}")
else :
    res = a + 8
    print(f"O número digitado é ímpar, somado com oito dá : {res}")